package.json

```diff
@@ -24,6 +24,7 @@
     "markdown-it": "^13.0.1",
     "mimetext": "^3.0.24",
     "next": "14.2.6",
+    "nhs-notify-web-template-management-ui-tests": "file:tests/test-team",
```

Comment:
what's the benefit of having this here? it seems like it will install all the test-related stuff whenever we do an npm install, including for production builds, which i don't think is ideal